.github/workflows/ci-fix-bot.yml

```
@@ -206,179 +206,29 @@ jobs:
-                  `_Auto-applied by **GARVIS CI Fix Bot**._`,
-                ].join('\n'),
-                labels: ['ci-fix-bot', 'auto-fix'],
-              });
-
-              core.info(`Created fix PR #${pr.data.number}`);
-            } catch (e) {
-              core.error(`Failed to create fix PR: ${e.message}`);
-            }
-
-      - name: Create issue for unfixable failures
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)